zerver/views/users.py

```diff
@@ -229,13 +229,22 @@ def regenerate_bot_api_key(request, user_profile, email):
     )
     return json_success(json_result)
 
+def add_outgoing_webhook_service(name, user_profile, base_url, interface, token):
+    # type: (Text, UserProfile, Text, int, Text) -> None
+    Service.objects.create(name=name,
+                           user_profile=user_profile,
+                           base_url=base_url,
+                           interface=interface,
+                           token=token)
+
 @has_request_variables
 def add_bot_backend(request, user_profile, full_name_raw=REQ("full_name"), short_name=REQ(),
-                    bot_type=REQ(validator=check_int, default=UserProfile.DEFAULT_BOT),
+                    bot_type=REQ(validator=check_int, default=UserProfile.DEFAULT_BOT), service_base_url=REQ(),
```

Comment:
I think we could also add a validator function like "check_url" in /zerver/lib/validator.py file.

What is your opinion?